src/main/resources/templates/fragments.html

```diff
@@ -25,8 +25,10 @@
         <div class="collapse navbar-collapse">
             <div class="navbar-nav">
                 <a class="nav-link home-link" th:href="@{/}">Home</a>
-                <a class="nav-link" th:href="@{/gardens/list}" sec:authorize="isAuthenticated()">Water Gardens</a>
+                <a class="nav-link list-link" th:href="@{/gardens/list}" sec:authorize="isAuthenticated()">Water Gardens</a>
                 <a class="nav-link map-link" th:href="@{/map}" sec:authorize="isAuthenticated()">Map</a>
+                <a class="nav-link" th:href="@{/features/}" sec:authorize="isAuthenticated()">Features</a>
```

Comment:
I had removed these in my last pull request since they could be accessed through the /admin page now. Did you want to keep them in the navbar and hide from normal users? I wouldn't have a strong preference either way. We can discuss in class later if you want!